doc/secciones/01_intro.tex

```diff
@@ -31,9 +31,9 @@ \section{Definición del problema}
 
 Dentro de este marco de trabajo, para realizar una mejor de definición del problema, se han identificado una serie de personas con necesidades y objetivos específicos que junto con las historias de usuario describen los problemas que se pretenden resolver con este proyecto.
 
-La metodología utilizada ha sido la de `personas`. Esta técnica se basa en la representación ficticia, pero concreta del grupo de usuarios al que va dirigido nuestra solución. El fin de este análisis es el poder llevar a cabo una compresión clara de los objetivos y necesidades de los usuarios en contextos específicos de uso \cite{cooper2014face}.
+La metodología utilizada ha sido la de `personas`. Esta técnica se basa en la representación ficticia, pero concreta del grupo de usuarios al que va dirigido nuestra solución. El fin de este análisis es el poder llevar a cabo una compresión clara de los objetivos y necesidades de los usuarios en contextos específicos de uso~\cite{cooper2014face}.
 
-\subsection{Usuarios identificados}
+\subsection{Usuarios identificados}~\label{sec:usuarios_identificados}
```

Comment:
Si metéis en un PR cosas irrelevantes al mismo, es más complicado de revisar... Convendría que estas cosas las hiciérais antes o después.

Reply:
La próxima vez guardaré una nota para cambiar estas cosas a _posteriori_.